src/vs/workbench/contrib/interactiveSession/browser/actions/interactiveSessionAccessibilityHelp.ts

```diff
@@ -0,0 +1,25 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { localize } from 'vs/nls';
+import { format } from 'vs/base/common/strings';
+import { IKeybindingService } from 'vs/platform/keybinding/common/keybinding';
+
+export function getAccessibilityHelpText(keybindingService: IKeybindingService): string {
+	const content = [];
+	content.push(localize('interactiveSession.accessibilityHelp', "Interactive Session Accessibility Help"));
+	content.push(localize('interactiveSession.helpMenuExit', "Exit this menu and return to the interactive editor input via the Escape key."));
+	content.push(descriptionForCommand('interactiveSession.action.focus', localize('interactiveSession.action.focus', 'The Focus Interactive Session ({0}) command focuses the chat request/response list, which can be navigated with UpArrow/DownArrow.'), localize('interactiveSession.action.focusNoKb', 'The Focus Interactive Session command focuses the chat request/response list, which can be navigated with UpArrow/DownArrow and is currently not triggerable by a keybinding.'), keybindingService));
```

Comment:
Missing `{0}` parameter?

Reply:
no - a bit confusing, but that get's set in `descriptionForCommand`
![Screenshot 2023-05-12 at 2 40 11 PM](https://github.com/microsoft/vscode/assets/29464607/6ce2def5-ad0f-43e2-a0f8-45007e9fb521)